app/assets/stylesheets/avalon.scss

```diff
@@ -1013,6 +1013,36 @@ div.structure_edit {
   visibility: visible !important;
 }
 
+/* Playlist index page */
+#playlists_top {
+  text-align: right;
+}
+
+#Playlists_info {
+  float: left;
+}
+
+#Playlists_length {
+  display: inline-block;
+}
+
+#Playlists_paginate {
+  float: right;
+}
+
+#playlist_filter_container {
+  display: inline-block;
+  margin-left: 1em;
+}
+
+#tag_filter_container {
+  display: inline-block;
+}
+
+#tag_filter {
+  display: inline;
+  width: auto;
+}
```

Comment:
Just a note, it's generally best practice to target `class`, instead of `id` in CSS.  A lot of code quality tools (like Hound for example), will complain.